aptos-move/move-examples/token_objects/guild/Move.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = 'guild'
+version = '1.0.0'
+
+[addresses]
+token_objects = "_"
```

Comment:
Can you name this guild instead? Let's keep separate names for separate packages.

Reply:
Done.